pkgs/misc/cups/drivers/brother-9020cdw/default.nix

```
@@ -0,0 +1,85 @@
+{ lib
+, stdenv
+, fetchurl
+, cups
+, dpkg
+, gnused
+, makeWrapper
+, ghostscript
+, file
+, a2ps
+, coreutils
+, perl
+, gnugrep
+, which
+, gawk
+}:
+
+let
+  version = "1.1.2-1";
+  lprdeb = fetchurl {
+    url = "https://download.brother.com/welcome/dlf100441/dcp9020cdwlpr-${version}.i386.deb";
+    sha256 = "1z6nma489s0a0b0a8wyg38yxanz4k99dg29fyjs4jlprsvmwk56y";
+  };
+
+  cupsdeb = fetchurl {
+    url = "https://download.brother.com/welcome/dlf100443/dcp9020cdwcupswrapper-${version}.i386.deb";
+    sha256 = "04yqm1qv9p4hgp1p6mqq4siygl4056s6flv6kqln8mvmcr8zaq1s";
+  };
+
+  model = "dcp9020cdw";
+
+in
+stdenv.mkDerivation {
+  name = "cups-brother-${model}";
```

`pname` + `version` is recommended here, you may check https://github.com/NixOS/nixpkgs/issues/103997 for more details.

```suggestion
  pname = "cups-brother-${model}";
  inherit version;
```

---

Ok, I'll try to change that.